src/apps/loyalty/server/middlewares/relay.ts

```diff
@@ -1,7 +1,7 @@
 import { NextFunction, Request, Response } from "express"
 import * as Relay from "react-relay"
 
-export const RelayMiddleware = (req: Request, res: Response, next: NextFunction) => {
+export default (req: Request, res: Response, next: NextFunction) => {
```

Comment:
I would prefer that we use named function syntax where possible, because that aids in debugging (named functions get identified as such).